kdl_parser/test/test_inertia_rpy.cpp

```diff
@@ -1,40 +1,70 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
This should be Open Source Robotics Foundation 2015 (https://github.com/ros/kdl_parser/commits/kinetic-devel/kdl_parser/test/test_inertia_rpy.cpp)

Reply:
Yeah, it looks like Jackie wrote that file initially (with no copyright notice), and then @Karsten1987 added the copyright notice earlier this year.  So I agree with you there, and I've changed it.  It still doesn't pass copyright checks, so I disabled it in 4c440e8